pkg/controller/networkpolicy/crd_utils.go

```diff
@@ -177,15 +177,16 @@ func (n *NetworkPolicyController) toAntreaPeerForCRD(peers []v1alpha1.NetworkPol
 		} else if peer.NodeSelector != nil {
 			addressGroup := n.createAddressGroup("", nil, nil, nil, peer.NodeSelector)
 			addressGroups = append(addressGroups, addressGroup)
-		} else if peer.Scope == v1alpha1.ScopeClusterSet {
-			clusterSetScopeSelectors = append(clusterSetScopeSelectors, antreatypes.NewGroupSelector(np.GetNamespace(), peer.PodSelector, peer.NamespaceSelector, nil, nil))
```

Comment:
Does it mean it will create AddressGroup anyway no matter it's ClusterSet scope or not?
I am wondering how we can verify that the stretchedNetworkPolicy being spitted into two parts of NP correctly? Shall we mention this on the document?

Reply:
> Does it mean it will create AddressGroup anyway no matter it's ClusterSet scope or not?
If there are any pod/nsSelectors, yes it will create AddressGroups because an AddressGroup will need to be created no matter the scope of these selectors.

> I am wondering how we can verify that the stretchedNetworkPolicy being spitted into two parts of NP correctly? 
E2E testcases should verify this.